src/upload/src/interface.ts

```diff
@@ -124,7 +124,7 @@ export interface XhrHandlers {
 
 export interface UploadInst {
   openOpenFileDialog: () => void
-  submit: () => void
+  submit: (fileId?: string) => void
```

Comment:
How is this change related?

Reply:
I changed the title and consolidated all the fixes into one PR to avoid making another.